local-kubeconfig

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
This file should be removed & added to `.gitignore`. I'll update the template to make this note.